arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,440 @@
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database that has the replication link to be dropped."
+                    },
+                    {
+                        "name": "linkId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The id of the replication link to be deleted."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": ""
+                    },
+                    "204": {
+                        "description": ""
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "DatabaseReplicationLinks"
+                ],
+                "operationId": "Databases_GetReplicationLink",
+                "description": "Returns information about an Azure SQL Database Replication Link.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database to get the link for."
+                    },
+                    {
+                        "name": "linkId",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The replication link id to be retrieved."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ReplicationLink"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/replicationLinks/{linkId}/failover": {
+            "post": {
+                "tags": [
+                    "DatabaseReplicationLinks"
+                ],
+                "operationId": "Databases_FailoverReplicationLink",
+                "description": "Begins failover of the Azure SQL Database Replication Link with the given id.",
```

Comment:
I am not aware of a way to cancel it.

---
In reply to: [80305533](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80305533) [](ancestors = 80305533)